pkg/notes/document/document.go

```diff
@@ -328,7 +364,9 @@ func sortKinds(notesByKind NotesByKind) []Kind {
 
 // CreateDownloadsTable creates the markdown table with the links to the tarballs.
 // The function does nothing if the `tars` variable is empty.
-func CreateDownloadsTable(w io.Writer, bucket, tars, prevTag, newTag string) error {
+//
+// Deprecated: Prefer using the golang template instead of markdown. Will be removed in #1019
+func CreateDownloadsTable(w io.Writer, bucket, tars, prevTag, newTag string) error { // Does this /need/ to be public?
```

Comment:
Yes, this needs to be public. :) we download the release notes from a remote location for a new k8s minor release, which means that we have to create the downloads table during the release build (see changelog.go).

Reply:
Acknowledged. Removed the comments.